packages/core/src/sync-realtime/service.test.ts

```diff
@@ -39,6 +37,7 @@ test("setup() returns block numbers", async (context) => {
   expect(finalizedBlockNumber).toBe(blockNumbers.finalizedBlockNumber);
 
   await service.kill();
+  networks[0].requestQueue.clear();
```

Comment:
Why is this required?

Reply:
Resets the timing on the queue so the next test doesn't have to wait for the time buffer